src/helpers/DockerHelper.js

```diff
@@ -8,38 +8,64 @@ export default class DockerHubHelper {
   }
 
   static doesImageExistWithCallback(image, callback) {
-    const [imageName, imageTag] = image.split(':')
-    const client = drc.createClientV2({
-      name: imageName
-    })
+    let [imageName, imageTag] = image.split(':')
     if (!imageTag || imageTag === 'latest') {
-      client.close()
       callback(null, false)
     } else {
-      client.getManifest({ ref: imageTag }, function(err, manifest) { // eslint-disable-line no-unused-vars
-        client.close()
-        callback(null, !err)
+      DockerHubHelper.getToken(imageName, function(error, token) {
+        if (error) {
+          callback(error, null)
+        } else {
+          if (!imageName.includes('/')) [
```

Comment:
```suggestion
          if (!imageName.includes('/')) {
```
right?  Also, is library in the next line part of the api url?

Reply:
library is the "default" org, and is needed if the image does not contain an org. (for example, our repos, like "urbancode/ucv-ext-jira", have an org. Other repos, like "mongo" do not have a repo, but in the API url, the org is required, so the "default" org of "library" needs to be added